src/components/ScriptEditor.jsx

```diff
@@ -185,9 +186,24 @@ class ScriptEditor extends React.Component {
     );
     this.setState({ editorState: newEditorState }, this.focus);
   }
+  
+
+
+  // renderAdminTextPreview() {
+  //   // console.log("renderAdminTextPreview in ScriptEditor");
+  //   const allData = this.state
+  //   // console.log("All Data", allData)
+  //     return (
 
+  //       <div>
+  //         <AdminTextPreview campaignProps={allData}/>
+  //       </div>
+  //       // <div style={styles.adminTextPreviewSection}></div>
+  //     );
+  // }
```

Comment:
Most of my time went into make this function work.

It was from a previous iteration where I thought the sampleScript could easily upload right above the text that was being created with the chips below.

Seeing it connected to customFields was probably why I focused to hard on this component.  But I also can't pass it all the information needed.  This is too far from the source, which ended up being AdminCampaignEdit